ethereum_history_api/oracles/src/fixtures/historyAPIConfig.ts

```diff
@@ -3,6 +3,7 @@ import { Address, Hex } from 'viem';
 export const JS_FIXTURES_DIRECTORY = 'fixtures';
 export const CIRCLE_USDC_BALANCE_STORAGE_KEY = '0x57d18af793d7300c4ba46d192ec7aa095070dde6c52c687c6d0d92fb8532b305';
 export const UNISWAP_V3_USDC_BALANCE_STORAGE_KEY = '0x1f21a62c4538bacf2aabeca410f0fe63151869f172e03c0e00357ba26a341eff';
+export const BORED_APE_7_HOLDER_STORAGE_KEY = '0x405787fa12a823e0f2b7631cc41b3ba8828b3321ca811111fa75cd3aa3bb5add';
```

Comment:
```suggestion
export const BORED_APE_7_OWNER_OF_STORAGE_KEY = '0x405787fa12a823e0f2b7631cc41b3ba8828b3321ca811111fa75cd3aa3bb5add';
```
If this is in fact the the `ownerOf` key

Reply:
7TH

Reply:
Done